JavaScript/Popup.js

```diff
@@ -0,0 +1,189 @@
+    'link to live version':
+            '#/',
+    'link to source':
+            '#',
+    id: 2,
+  },
+  {
+    name: 'Multi-Post Stories',
+    title: 'Uber Navigation',
+    description:
+            'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'featured image': './images/tonic-img-1.png',
+    'featured image desk': './images/uber-nav-desktop.svg',
+    technologies: ['html', 'css', 'javascript'],
+    'link to live version':
+            '#/',
+    'link to source':
+            '#',
+    id: 3,
+  },
+];
+ProjectInformation.forEach((element, i) => {
+  const work = document.querySelector('.card');
+  const card = document.createElement('article');
+  card.classList = 'card-section';
+  if (i === 1) {
+    card.setAttribute('id', 'reverce');
+  } else if (i === 3) {
+    card.setAttribute('id', 'reverce');
+  }
+
+  const popupcard = `
+    <img src="${ProjectInformation[i]['featured image']}" alt="Snapshot Portfolio" class="tonic-img-mobile">
+  <img src="${ProjectInformation[i]['featured image desk']}" alt="Snapshot Portfolio"
+      class="tonic-img-desktop">
+  <article class="card-section-text">
+  <h2 class="project-title-heading text-mobile">${ProjectInformation[i].name}</h2>
+  <h2 class="project-title-heading text-desktop">${ProjectInformation[i].title}</h2>
+  
+      <article class="card-sub-container">
+          <h2 class="title-canopy text-mobile">CANOPY</h2>
+          <h2 class="title-canopy text-desktop">CANOPY</h2>
+          <img src="./images/Counter.png" alt="Counter">
+          <h2 class="title-developer text-mobile">Back End Dev</h2>
+          <h2 class="title-developer text-desktop">Back End Dev</h2>
+          <img src="./images/Counter.png" alt="Counter">
+          <h2 class="title-year text-mobile">2015</h2>
+          <h2 class="title-year text-desktop">2015</h2>
+      </article>
+      <p class="desciption-p-mobile text-mobile">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <p class="desciption-p-mobile text-desktop">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="pro-language">
+      <li>
+      <h1 class="html-1">html</h1>
+  </li>
+  <li>
+  <h1 class="html-1" id="rube-desktop">Rube on rails</h1>
+</li>
+  <li>
+      <h1 class="html-1">css</h1>
+  </li>
+  <li>
+      <h1 class="html-1">Javascript</h1>
+  </li>
+      </ul>
+      <div class="btn-container">
+          <button class="btn-see-project" type="button" id="${ProjectInformation[i].id}">
+              See project
+          </button>
+      </div>
+  </article>
```

Comment:
**_To match the Figma design kindly fix these issues for the mobile version of the popup :_**

- [ ] the popup should be centered in the page kindly fix that 🙏 
- [ ] kindly add the blurry background behind the pop-up
- [ ] kindly add space between the action buttons `See Live & See Source` you can use flex and justify-around/between, and please add the button interaction styling such as Hover, press, etc ...
- [ ] your card missing the section below the title kindly add it 🙏 

Figma           | Yours
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/216044017-19db004b-582e-4971-a347-c600b931d63b.png) | ![image](https://user-images.githubusercontent.com/89282221/216044845-f4cae777-72fc-4c49-8c30-3c072ea0dfbc.png)